back/app/policies/idea_policy.rb

```diff
@@ -67,16 +66,12 @@ def draft_by_phase?
   def update?
     return false if !record.participation_method_on_creation.supports_edits_after_publication? && record.published? && !record.will_be_published?
     return true if (record.draft? && owner?) || (user && UserRoleService.new.can_moderate_project?(record.project, user))
-    return false unless active? && owner? && ProjectPolicy.new(user, record.project).show?
```

Comment:
Project visibility was already covered by the project permission service (which is "called" by the idea permission service).